test/execution-tests/3.0.1_resolveModuleName/test/app.tests.ts

```diff
@@ -0,0 +1,13 @@
+import * as app from '../src/app';
+
+// We don't actually care about the result of the following operations;
```

Comment:
This is not true actually; we *do* care about the result of the following operations!  Can we amend the comment to make test intent clearer please?  We care both about type resolution and that the code executes...